helm/publisher/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: publisher
+description: Libero Publisher vanilla flavor
+type: application
+version: 0.1.0
+appVersion: latest
```

Comment:
this is meaningless at the moment - sometimes used to select the Docker image tag but not the case here